src/components/MerchantsPage/gam/NoCampaignsBox.tsx

```diff
@@ -4,37 +4,23 @@ import {
   tabletScreens,
 } from '../../../utilities/general/responsive';
 import styled from 'styled-components';
-import Modal from '../../Modal';
-import { useModalPaymentDispatch } from '../../../utilities/hooks/ModalPaymentContext/context';
-import { SET_MODAL_VIEW } from '../../../utilities/hooks/ModalPaymentContext/constants';
 import { useTranslation } from 'react-i18next';
 
-const ModalBox: any = Modal;
-
 const NoActiveCampaignsBox = () => {
   const { t } = useTranslation();
 
-  const dispatch = useModalPaymentDispatch(); //provide null according to Bruce's new branch
-  const showModal = (event: any) => {
-    dispatch({ type: SET_MODAL_VIEW, payload: 0 });
-  };
-
   return (
     <NoCampaignBox>
       <TextContainer>
         <Heading>{t('gamHome.noCampaignsBox.CTA')}</Heading>
         <SubHeading>{t('gamHome.noCampaignsBox.description')}</SubHeading>
       </TextContainer>
-      <Button className="button--filled" onClick={showModal}>
+      <Button className="button--filled" onClick={(e) => {
+          e.preventDefault();
+          window.open('https://www.gofundme.com/f/gift-a-meal', '_blank')
```

Comment:
is it possible to open this in a separate tab so that people can quickly go back to our website?

Reply:
Yes, this currently opens into a new tab!